.github/workflows/unit_tests_cloud.yml

```diff
@@ -0,0 +1,153 @@
+# This workflow will install Python dependencies, and run a selection of tests from the Odemis repository.
+# The selection is picked such that they do not require a back-end and are known to pass with this github workflow.
+# Test cases that will pass with this workflow can be added to this document.
+# It uses the python 3 system interpreter to run test cases.
+
+name: Unit testing
+
+on:
+  [push, pull_request]
+jobs:
+  build:
+
+    runs-on: ubuntu-20.04
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Install Odemis dependencies
+      run: |
+        sudo add-apt-repository ppa:delmic-soft/odemis
+        sudo apt-get -y update
+
+        sudo apt-get -y install python3-pil python3-yaml python3-serial python3-zmq \
+        python3-scipy python3-decorator \
+        python3-libtiff python3-comedilib python3-h5py libtiff5-dev python3-future \
+        python3-pkg-resources python3-netifaces python3-cairo python3-numpy python3-scipy \
+        python3-decorator python3-xdg python3-suds python3-matplotlib python3-opencv \
+        python3-wxgtk4.0 python3-serial python3-evdev python3-notify2 python3-psutil \
+        graphicsmagick cgroup-tools authbind python3-setuptools cython python3-pyro4-delmic
+
+        sudo mkdir /var/run/odemisd
+        sudo chmod a+rw /var/run/odemisd
+
+    - name: Run tests from odemis.dataio
+      if: success() || failure()  # run this step if the last step passed or failed, but not if it was canceled
```

Comment:
I meant the previous step. The difference between this and `continue-on-error` is that this will report a fail for the entire run if one of the steps fails and `continue-on-error` will report that the entire run passed. I'll update the comment to explain